dev-vcs/tig/tig-2.2.1.recipe

```diff
@@ -46,11 +55,23 @@ BUILD()
 {
 	libtoolize --force --copy --install
 	./autogen.sh
-	runConfigure ./configure
+	runConfigure --omit-dirs binDir ./configure --bindir=$commandBinDir
 	make
 }
 
 INSTALL()
 {
 	make install
+
+	# temporary solution because of xmlto deps failing
```

Comment:
You have convinced me ;-)
That said, if anyone disagrees or has an easy workaround for the issues found on the offending docs that fail docbook validation, please feel free to jump into this discussion!

So let's ignore my previous comment about using **`cmd:xmlto`**.
(Once you remove the last empty line we will be able to merge this PR, if nobody complains ;-)